compiler/ir/stream/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .access_pattern import *
+from .scheduler import *
```

Comment:
For my information: Is this an import helper?

Reply:
this way you can do 
```
from compiler.ir.stream import Template
```
instead of having to do
```
from compiler.ir.stream.access_pattern import Template
```